Changelog.md

```diff
@@ -1,3 +1,7 @@
+Unreleased
+-------------------------
+* [Chore] Drop support for Python 3.8 and `XBlock<2`.
```

Comment:
I don't know exactly how to categorise this but I think it's more than a chore. Maybe just go with this:

```suggestion
* Drop support for Python 3.8 and `XBlock<2` (and, as a consequence, any Open edX releases prior to Redwood).
```